app/javascript/gobierto_data/lib/helpers.js

```diff
@@ -0,0 +1,11 @@
+function getToken() {
```

Comment:
Can't we do `window.gobiertoAPI.token`?

Reply:
Somebody kill me. `window` !! 😱

Reply:
Haha, for the moment is fine, but let's propose another way ;)

Reply:
I just read your review and got scared. I didn't realize this function is getting the token from the HTML body, what it's against any sort of security rule.

I presume that's temporary so, yep, you may do something like that, or actually do nothing because this is gonna be removed eventually.